supply/lib/supply/uploader.rb

```diff
@@ -195,7 +195,13 @@ def upload_changelog(language, version_code)
         UI.message("Updating changelog for '#{version_code}' and language '#{language}'...")
         changelog_text = File.read(path, encoding: 'UTF-8')
       else
-        UI.message("Could not find changelog for '#{version_code}' and language '#{language}' at path #{path}...")
+        path = File.join(Supply.config[:metadata_path], language, Supply::CHANGELOGS_FOLDER_NAME, "default.txt")
+        if File.exist?(path)
+          UI.message("Updating default changelog for '#{version_code}' and language '#{language}'...")
```

Comment:
```suggestion
          UI.message("Updating changelog for '#{version_code}' and language '#{language}' to default changelog...")
```

Reply:
I wanted I slightly different message so you can know whether it used the default changelog or not, what do you think about keeping that?

Reply:
Yeah, but what happens here is that the "changelog" is updated. In this new case it is "update to the default changelog text" - hence my suggestion.

Reply:
Oh my bad I didn’t notice the text added at the end of the line! Looks good.